scripts/build-bitcoin-images-k8s.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Create a new builder to enable building multi-platform images
+docker buildx create --use
+
+# Image and Registry info
+DOCKER_REGISTRY="bitcoindevproject/k8s-bitcoin-core"
+REPO="bitcoin/bitcoin"
+BUILD_ARGS="--disable-tests --with-incompatible-bdb --without-gui --disable-bench --disable-fuzz-binary --enable-suppress-external-warnings --without-miniupnpc --without-natpmp"
+echo "DOCKER_REGISTRY=${DOCKER_REGISTRY}"
+echo "REPO=${REPO}"
+
+# Tags and their supported architectures
+declare -A VERSION_ARCH_MAP=(
+    ["23.2"]="amd64 arm64 armhf"
+    ["24.2"]="amd64 arm64 armhf"
+    ["25.1"]="amd64 arm64 armhf"
+)
+
+if [[ -d "src/templates" ]]; then
+  cd src/templates || exit 1
+else
+  echo "Directory src/templates does not exist. Please run this script from the project root."
+  exit 1
+fi
+
+# Loop through each tag and its architectures to build and push
+for VERSION in "${!VERSION_ARCH_MAP[@]}"; do
+    IFS=' ' read -ra ARCHS <<< "${VERSION_ARCH_MAP[${VERSION}]}"
+    IMAGES_LIST=()  # Array to store images for manifest
+    for DOCKER_ARCH in "${ARCHS[@]}"; do
+        echo "BRANCH=v${VERSION}"
+        echo "DOCKER_ARCH=${DOCKER_ARCH}"
+
+        IMAGE_TAG="${VERSION}-${DOCKER_ARCH}"
+        IMAGE_FULL_NAME="${DOCKER_REGISTRY}:${IMAGE_TAG}"
+        echo "IMAGE_FULL_NAME=${IMAGE_FULL_NAME}"
+
+        # Use Buildx to build the image for the specified architecture
+        docker buildx build --platform linux/"${DOCKER_ARCH}" \
+            --provenance=false \
+            --build-arg REPO="${REPO}" \
+            --build-arg BRANCH="v${VERSION}" \
+            --build-arg BUILD_ARGS="${BUILD_ARGS}" \
+            --tag "${IMAGE_FULL_NAME}" \
+            . --push
```

Comment:
This was not specifying `--file Dockerfile_k8s` so (for me) it was using the default `Dockerfile`.

This could be why @josibake observed networking not working? (perhaps patch not applied?)

Reply:
Could have been. I might have built and pushed the images and then renamed the Dockerfile but forgot to update it here. I might be misremembering but I thought I saw bitcoind compiling.

Reply:
We should re-push the images though as there is doubt.